app/models/vm_or_template.rb

```diff
@@ -1923,6 +1923,10 @@ def validate_supported
     {:available => true,   :message => nil}
   end
 
+  def validate_unsupported(message_prefix)
+    {:available => false, :message => "#{message_prefix} is not available for #{self.class.model_suffix} VM."}
```

Comment:
This message should change to not be `"VM"` specific.  It should say: 

> `"#{message_prefix} is not available for #{self.class.model_suffix} VM or Template."`

But, even then, we have the problem of cloud vs. infra (i.e., template vs image).  Change it to `"VM or Template"` for now.  We can deal with verbiage later.